examples/python-guide/plot_example.py

```diff
@@ -37,7 +37,7 @@
                 lgb_train,
                 num_boost_round=100,
                 valid_sets=[lgb_train, lgb_test],
-                feature_name=['f' + str(i + 1) for i in range(X_train.shape[-1])],
+                feature_name=[f'f {i+1}' for i in range(X_train.shape[-1])],
```

Comment:
```suggestion
                feature_name=[f'f{i+1}' for i in range(X_train.shape[-1])],
```

I think this might be the cause of the test failures you saw. `+` does not add any spaces, but you've added one here.

Reply:
I will fix this right away, thanks a lot!

Reply:
thanks! In the future when you work on GitHub and see one of these "suggested change" thing, you will probably find it easier to click "Commit suggestion" in the browser. That automatically commits the change and resolves the comment thread.

Reply:
Will do that, thanks sir!